project/Dependencies.scala

```diff
@@ -52,7 +52,7 @@ object Dependencies {
       * Runtime errors occur when the versions are incompatible.
       */
     val pubsubFs2 = "0.15.0"
-    val scio      = "0.9.4"
+    val scio      = "0.9.6"
```

Comment:
Have you successfully deployed and run any rc using scio 0.9.6?

If you can't because of an error message like `Caused by: java.lang.ClassNotFoundException: io.netty.internal.tcnative.SSLPrivateKeyMethod` that's because scio 0.9.6 doesn't come with `io.netty:netty-tcnative-boringssl-static:2.0.30.Final`. Just add it explicitly on your own and it should be fine. e.g. [beam enrich's fix](https://github.com/snowplow/enrich/commit/0dc7e49ea88a8902ae3e554b344ea302b6e4f28e)

Reply:
It has been running fine. Would it fail on startup or only under certain conditions?